just_audio/darwin/Classes/AudioPlayer.m

```diff
@@ -1330,6 +1330,15 @@ - (void)dispose {
     if (!_player) return;
     if (_processingState != none) {
         [_player pause];
+
+        [self updatePosition];
+        [self broadcastPlaybackEvent];
```

Comment:
Facing this issue the app crashes when I close it. could it be because of this line change?
-[AudioPlayer broadcastPlaybackEvent]
NSInternalInconsistencyException - Sending a message before the FlutterEngine has been run.